### DISCUSSION TITLE
Add tool to draw a straight line (really)

### DISCUSSION THREAD

**Discussion by bar-niedo** [OpenOrienteering/mapper#2151] [Ideas]
The toolbar could be improved.
Among other things, it would be nice if new users didn't have to spend 10 minutes searching for a way to draw a SIMPLE STRAIGHT LINE.

**Comment by dg0yt** [comment]
Please make a concrete suggestion.

The current situation situation is that the line drawing tool does all type of lines, straight lines by default, and it is the tool which is automatically activated when you select a new line symbol.

**Comment by mlerjen** [comment]
Welcome to Mapper and enjoy... 
[Mapper drawing tutorial.zip](https://github.com/OpenOrienteering/mapper/files/11293627/Mapper.drawing.tutorial.zip)
